src/extension/wps/src/main/java/au/org/emii/aggregator/NetcdfAggregator.java

```diff
@@ -120,8 +123,9 @@ private void assertOutputPathValid(Path outputPath) {
 
     private void copyToOutputFile(NetcdfDatasetIF template) throws AggregationException {
         try {
-            writer = NetcdfFileWriter.createNew(NetcdfFileWriter.Version.netcdf4,
-                outputPath.toString(), null);
+            Nc4Chunking chunking = new Nc4ChunkingDefault(3, false);
```

Comment:
Yes it does and I did consider it.  You could conceivably need/want to configure it at the variable, dataset and/or global level.  This would add a bit of complexity/work which would blow out the scope of this change.